<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Critical Alert Management Console</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>mainICo.png</normaloff>mainICo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    color: #eaeaea;
    background-color: #2b2b2b;
    selection-background-color: #3daee9;
    selection-color: #eaeaea;
}

QHeaderView::section {
    background-color: #3b3b3b;
    color: #eaeaea;
    padding: 5px;
}

QTabWidget::pane {
    border: 1px solid #222222;
    background-color: #2b2b2b;
}

QTabBar::tab {
    background-color: #2b2b2b;
    color: #eaeaea;
    padding: 8px;
    margin-right: 1px;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background-color: #3daee9;
    color: #eaeaea;
}

QLineEdit, QTextEdit, QPlainTextEdit, QListView, QTableView, QTreeView {
    background-color: #333333;
    color: #eaeaea;
    selection-background-color: #3daee9;
    selection-color: #eaeaea;
    border: 1px solid #555555;
    border-radius: 2px;
    padding: 2px;
}

QProgressBar {
    border: 1px solid #222222;
    border-radius: 5px;
    background-color: #555555;
    color: #eaeaea;
}

QProgressBar::chunk {
    background-color: #3daee9;
    width: 5px;
    margin: 0.5px;
}

QPushButton {
    color: #eaeaea;
    background-color: #333333;
    border: 1px solid #555555;
    padding: 5px;
    border-radius: 2px;
}

QPushButton:hover {
    background-color: #3daee9;
    color: #eaeaea;
}

QToolButton {
    color: #eaeaea;
    background-color: #333333;
    border: 1px solid #555555;
    padding: 5px;
    border-radius: 2px;
}

QToolButton:hover, QToolButton:checked {
    background-color: #3daee9;
    color: #eaeaea;
}

QComboBox {
    color: #eaeaea;
    background-color: #333333;
    border: 1px solid #555555;
    padding: 2px;
    border-radius: 2px;
    min-width: 6em;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: #555555;
    border-left-style: solid;
}

QComboBox::down-arrow {
    image: url(arrow-down.png);
    width: 10px;
    height: 10px;
}

QComboBox::down-arrow:hover {
    image: url(arrow-down-hover.png);
}

QScrollBar {
    border: 1px solid #222222;
    background-color: #2b2b2b;
}

QScrollBar:horizontal {
    height: 15px;
    margin: 3px 15px 3px 15px;
}

QScrollBar:vertical {
width: 15px;
margin: 15px 3px 15px 3px;
}

QScrollBar::handle {
background-color: #555555;
border-radius: 5px;
}

QScrollBar::handle:hover {
background-color: #3daee9;
}

QScrollBar::add-page, QScrollBar::sub-page {
background-color: #2b2b2b;
}

QScrollBar::add-line, QScrollBar::sub-line {
border: 1px solid #222222;
background-color: #555555;
}

QCheckBox, QRadioButton {
color: #eaeaea;
}

QCheckBox::indicator, QRadioButton::indicator {
width: 15px;
height: 15px;
border: 1px solid #555555;
border-radius: 2px;
background-color: #333333;
}

QCheckBox::indicator:disabled, QRadioButton::indicator:disabled {
border-color: #777777;
background-color: #444444;
}

QCheckBox::indicator:checked, QRadioButton::indicator:checked {
background-color: #3daee9;
border-color: #3daee9;
}

QMenuBar {
background-color: #2b2b2b;
color: #eaeaea;
}

QMenuBar::item {
padding: 2px 10px 2px 10px;
background-color: transparent;
}

QMenuBar::item:selected {
background-color: #3daee9;
color: #eaeaea;
}

QMenu {
background-color: #2b2b2b;
border: 1px solid #222222;
}

QMenu::item {
background-color: transparent;
padding: 2px 20px 2px 20px;
}

QMenu::item:selected {
background-color: #3daee9;
color: #eaeaea;
}

QTreeView::item:selected {
background-color: #3daee9;
color: #eaeaea;
}

QTreeView::item:hover {
background-color: #3daee9;
color: #eaeaea;
}

QHeaderView::section:pressed {
background-color: #3daee9;
}

QTableView::item:selected, QListView::item:selected, QTreeView::item:selected {
background-color: #3daee9;
color: #eaeaea;
}

QTableView::item:hover, QListView::item:hover, QTreeView::item:hover {
background-color: #3daee9;
color: #eaeaea;
}</string>
  </property>
  <widget class="QPushButton" name="boardConnectButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserStatus">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>470</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>Status Infomations</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="placeholderText">
    <string>Status Bar</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="sendTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>451</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Good Practice : Connect Before Type!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red</string>
   </property>
   <property name="text">
    <string>Caution: Access to this system is a privilege. Please use it responsibly for its intended purposes to avoid endangering lives and property.</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Type Your Message Here:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>main.png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
